docs/extensibility/form-section.md

```diff
@@ -246,7 +246,7 @@ Name widgets render a name input field. They contain an automatic name generator
 
 - **extraPaths** - an array of extra paths to fill in with the contents of the field. Each path can either be a period-separated string or an array of strings.
 - **placeholder** - specifies a short hint about the input field value.
-- **inputInfo** - a string below the input field that shows how to fill in the input. It defaults to `Name must consist of lowercase alphanumeric characters, can contain '-' and '.' (e.g.: 'my.name-1').`. To disable any suggestion, set this value to `null`.
+- **inputInfo** - a string below the input field that shows how to fill in the input. It defaults to `Name must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character (e.g. 'my-name', or '123-abc').`. To disable any suggestion, set this value to `null`.
```

Comment:
```suggestion
- **inputInfo** - a string below the input field that shows how to fill in the input. It defaults to `Name must consist of lowercase alphanumeric characters or '-', and must start and end with an alphanumeric character (e.g. 'my-name', or '123-abc').`. To disable any suggestion, set this value to `null`.
```

```suggestion
    k8s-name-input: "Name must consist of lowercase alphanumeric characters or '-', and must start and end with an alphanumeric character (e.g. 'my-name', or '123-abc')."
```

I think it should stay as it was. [K8s docs](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names) use `lowercase` instead of `lower case`, but it will be verified by TWs.

Reply:
Yes, let's keep `lowercase`.